### PR TITLE
Fix `npm test` error

### LINKED ISSUE #31: `npm test` crashes
> postcss-custom-selectors@3.0.0 lint E:\Dmitry\git\postcss\postcss-custom-selectors eslint .
>
> TypeError: Cannot read property 'visitClass' of undefined
    at monkeypatch (E:\Dmitry\git\postcss\postcss-custom-selectors\node_modules\babel-eslint\index.js:199:40)

Upgrade `babel-eslint` or (upgrade `eslint` and removing `babel-eslint`) will fix it.

### PR DESCRIPTION
**Pull request by Semigradsky**
Fix #31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,21 +24,19 @@
...
     "babel-tape-runner": "^1.1.0",
-    "eslint": "^1.0.0",
+    "eslint": "^3.15.0",
     "tape": "^4.0.0"
   },
   "scripts": {
     "prepublish": "babel src --out-dir dist",
     "lint": "eslint .",
-    "tape": "babel-tape-runner 'test/*.js'",
+    "tape": "babel-tape-runner test/*.js",
```
Comment (Semigradsky) [inline package.json]: With comma not working on Windows

File: test/index.js [hunk 2]
```diff
@@ -11,9 +11,9 @@ function transform(input, opts = {}, postcssOpts = {}) {
 test("@custom-selector", function(t) {
   t.equal(
     transform(
-      ``
+      ""
```
Comment (Semigradsky) [inline test/index.js]: Fix lint error

File: package.json [hunk 3]
```diff
@@ -24,21 +24,19 @@
     "README-zh.md"
   ],
   "dependencies": {
-    "balanced-match": "^0.2.0",
```
Comment (Semigradsky) [inline package.json]: Old dependency?

File: package.json [hunk 4]
```diff
@@ -24,21 +24,19 @@
...
     "postcss": "^5.0.0",
     "postcss-selector-matches": "^2.0.0"
   },
   "devDependencies": {
     "babel": "^5.5.8",
-    "babel-eslint": "^3.1.15",
```
Comment (Semigradsky) [inline package.json]: It's no longer necessary

**Comment by MoOx** [comment]
Thanks!

**Pull request merged**